azure_acquire/util.py

```diff
@@ -239,12 +239,14 @@ def capture_from_azure(k4a, filename_prefix, recording_length, save_ir=True,
     finally:
         k4a.stop()
         system_timestamps = np.array(system_timestamps)
```

Comment:
delete this line

Reply:
e927585faa7278b18e2abf2f07dcd4e2bbc7710f